apps/web/src/server/applications/case/documentation/applications-documentation.router.js

```diff
@@ -117,4 +119,10 @@ applicationsDocumentationRouter
 		asyncRoute(controller.viewApplicationsCaseDocumentationPages)
 	);
 
+console.log('I am here');
+applicationsDocumentationRouter
+	.route('/document-search-results/:pageNumber?')
```

Comment:
the full URL would become /applications-service/case/${caseId}/project-documentation/document-search-results/${pageNumber}.

is that what you want?